### ISSUE TITLE
Show retransmit/error count

### ISSUE THREAD

**Issue by winbatch** [ndfred/iperf-ios#56]
To know if slow speed is due to retransmits

**Comment by ndfred** [comment]
That is impossible because TCP handles retransmits transparently. Implementing UDP testing would give you that information, see #28 and #51. Closing as won't fix.

**Comment by winbatch** [comment]
Sorry, don't see what you mean, since this already exists on the windows and linux version.  And I doubt TCP operates differently there.

**Comment by ndfred** [comment]
You're right, I never really paid attention to the `retr` column and that would actually be a good addition. Had no idea TCP could give you that information. Re-opening the issue.
